src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -422,19 +422,23 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
 
     /*******************************************************************************************************
     * @description for the list of AccountId's, recalc their rollups 
-    * @param listAccountId the list of Account (Id's) that need updating.
+    * @param accountIds the list of Account (Id's) that need updating.
     * @return void
     */ 
-    public static void rollupAccounts(list<Id> listAccountId) {
-        if (listAccountId.size() == 1 || system.isFuture() || system.isBatch()) {
-            RLLP_OppRollup_UTIL oppRollupUtil = new RLLP_OppRollup_UTIL(); 
-            map<id, Account> amap = new map<id, account>((list<Account>)Database.query(oppRollupUtil.buildAccountQuery() + 
-                                                           ' where id IN :listAccountId')); 
-            RLLP_OppRollup rg = new RLLP_OppRollup();
-            rg.rollupAccounts(amap);                    
+    public static void rollupAccounts(List<Id> accountIds) {
```

Comment:
It looks like this method only works with legacy rollups. If that's the case, we should fix this so that it calls either Legacy or Customizable Rollups based on what's enabled. The `CON_ContactMerge_TDTM` class has logic that does this

Reply:
@force2b - I agree but I think it should be a separate work item.

Reply:
Fair enough. Can you create one for this?